raiden_contracts/contract_manager.py

```diff
@@ -216,7 +219,7 @@ def _load_json_from_path(f: Path):
     try:
         with f.open() as deployment_file:
             return json.load(deployment_file)
-    except (FileNotFoundError) as ex:
+    except FileNotFoundError as ex:
```

Comment:
if you do not need to acces sor use the exception, there's no need for `as ex`.

Reply:
I'll fix this in this PR.